app/controllers/catalog_controller.rb

```diff
@@ -268,9 +268,11 @@ def st_delete
     if params[:id]
       elements.push(params[:id])
       process_sts(elements, 'destroy') unless elements.empty?
-      add_flash(_("The selected %{record} was deleted") %
-        {:record => ui_lookup(:table => "service_template")}) if @flash_array.nil?
-      self.x_node = "root"
+      if @flash_array.nil?
+        add_flash(_("The selected %{record} was deleted") %
```

Comment:
I know this has been there before, but for the sake of translators and nicer translations, we'd like
to get rid of calling `ui_lookup()` on a static string. So I'd suggest to change the above string to:
`"The selected Catalog Item was deleted"`